discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown

```diff
@@ -0,0 +1,46 @@
+# Patch types [](id=patch-types)
+
+Liferay ships Digital Enterprise fixes through three different channels. Fix packs and hotfixes. You can read about the differences in this document.
```

Comment:
You say three channels but then list two: fix pack and hotfixes. Is there another one, or should it be two channels?